File: problem-solving/Easy/4_a-very-big-sum/javascript/main.js
// URL : https://www.hackerrank.com/challenges/a-very-big-sum

function aVeryBigSum(ar) {
  // Write your code here
  return [...ar]
    .filter((p) => 0 <= p && p <= Math.pow(10, 10))
    .reduce((acc, total) => {
      return acc + total;
    }, 0);
}

//test
console.log(aVeryBigSum([1, 2, 3]));

File: problem-solving/Easy/4_a-very-big-sum/python/main.py
# URL : https://www.hackerrank.com/challenges/a-very-big-sum
import math


def aVeryBigSum(ar):
    ar = [i for i in ar if 0 <= i <= math.pow(10, 10)]
    total = 0
    for i in ar:
        total += i
    return total


# test
print(aVeryBigSum([3, 4, 5]))
